packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -262,6 +287,13 @@ class Draggable<T extends Object> extends StatefulWidget {
   /// Where this widget should be anchored during a drag.
   final DragAnchor dragAnchor;
 
+  /// An optional injectable strategy that overrides the
+  /// [defaultDragAnchorStrategy] and is used to determine how this widget
+  /// should be anchored during a drag. When a custom [dragAnchorStrategy] is
+  /// injected it is asserted that the [dragAnchor] is custom, to prevent any
```

Comment:
there is no DragAnchor.custom?

Reply:
I feel with should deprecate DragAnchor and just uses the DragAnchorStrategy, and we can provide different DragAnchorStrategy(one for DragAnchor.child and one for DragAnchor.pointer)

Reply:
> there is no DragAnchor.custom?

Yes. That looks like a leftover from a work in progress state. Thanks for the heads up!

Reply:
Regarding deprecating DragAnchor. Yes, for the future I would also suggest that. But as a new contributor I did not want to make such a bold move and suggest that right away. First I wanted to show my alternative approach :-)